race.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import time
+
+from constants import *
+from prerace_setup import _initialize_rounds, _load_race_settings, _pre_round_preparation
+
+RACER_RUNTIME = 0
+RACER_NAME = 1
+
+
+def race():
+    race_settings = _load_race_settings()
+    rounds = _initialize_rounds(race_settings)
+
+    loop = asyncio.get_event_loop()
+    for curr_round in rounds:
```

Comment:
the logic of what happens in each round in inside the `race` function instead of its own one